.github/workflows/uitests.yml

```diff
@@ -0,0 +1,26 @@
+name: Run UI Tests
+
+on:
+  push:
+    paths:
+    - 'thea-ui/**'
+
+jobs:
+
+  ui-tests:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install npm dependencies
+      run: npm install --verbose
+      working-directory: ./thea-ui
+    
+    - name: Run linter
+      run: npm run lint
```

Comment:
I believe some files are failing the linter when I run this locally.

Reply:
Did you use the "pointless change" push to test the GitHub action or did you run it locally using a different means?

Reply:
I ran it locally with `npm run lint`.

Reply:
Is it failing because of trailing whitespace? A few of the `.spec` files have that issue.